DigitalLearningSolutions.Web/Views/Frameworks/Shared/_NavMenuItems.cshtml

```diff
@@ -1,5 +1,6 @@
 ﻿@using DigitalLearningSolutions.Web.Extensions
 <li class="nhsuk-header__navigation-item @Html.IsSelected("FrameworksDashboard")">
+  @* ReSharper disable once Mvc.ActionNotResolved *@
   <a class="nhsuk-header__navigation-link" asp-action="MyFrameworks">
```

Comment:
Could we actually fix these to have no issues? I think asp-action="FrameworksDashboard" asp-controller="Frameworks" should work?

Reply:
The point of the ServiceFilter is that it's generic and so can be used on any controller. That does mean it relies on there being an action called Index on any controller it's used with though

Reply:
I see - can we keep the comment there but fix the frameworks in that case? :)